rfc/0053-sdk3-crud.md

```diff
@@ -670,6 +670,85 @@ Throws
 - Undocumented
   - InvalidArgumentException (#3)
 
+#### LookupInAnyReplica
+
+Gets a document for a given id, leveraging both the active and all available replicas.
+This method follows the same semantics of LookupInAllReplicas (including the fetch from ACTIVE), but returns the first response as opposed to returning all responses.
+
+Signature
+
+```csharp
+ILookupInReplicaResult LookupInAnyReplica(string id, LookupInSpec[] specs, [LookupInAnyReplicaOptions])
+```
+
+Parameters
+
+- Required
+
+  - Id: string - the primary key.
+  - LookupInSpec - an array of fetch operations, see [LookupIn](#LookupIn)
+
+- Optional
+
+  - Timeout or timeoutMillis (int/duration) - the time allowed for the operation to be terminated. This is controlled by the client.
+  - Serializer - a custom serializer for converting the memcached JSON data to a native type.
+  - Boolean AccessDeleted maps to 0x04 otherwise omitted - private; used to access XAttrs from a deleted document
+
+Returns
+
+- A ILookupInReplicaResult wrapping a ILookupInResult object.
+
+Throws
+
+NOTE: The following exceptions are thrown at the "top" level. The errors that are thrown on a per-field basis are covered in [LookupIn](#LookupIn).
+
+- Throws
+  - Documented
+    - DocumentUnretrievableException (#125)
+      - The situation where we got responses from all of them (most likely: key not found) but none of them were SUCCESS so we ended up not returning anything (empty stream from lookupin all replicas).
+    - RequestTimeoutException (#1)
+    - CouchbaseException (#0)
+    - FeatureNotAvailableException (#15)
+  - Undocumented
+    - InvalidArgumentException (#3)
+
+#### LookupInAllReplicas
+
+Gets a stream of document data from the server using LookupIn, leveraging both the active and all available replicas; the caller can choose which replica they wish to take and cancel the stream at any time.
+This MUST fetch from both the active (using a standard GET opcode) and all replicas (via the usual GET_REPLICA opcode) due to failover situations that can arise.
```

Comment:
Are these references to the GET and GET_REPLICA opcodes relevant for subdoc read from replica?

Reply:
Now removed these references in the new RFC.